core/interfaces/i_accessibility.js

```diff
@@ -59,15 +59,14 @@ Blockly.IASTNodeLocationWithBlock.prototype.getSourceBlock;
 
 
 /**
- * An interface for an object that handles Blockly actions when keyboard
- * navigation is enabled.
+ * An interface for an object that handles keyboard shortcuts.
  * @interface
  */
-Blockly.IBlocklyActionable = function() {};
+Blockly.IKeyboardAccessible = function() {};
 
 /**
  * Handles the given action.
  * @param {!Blockly.ShortcutRegistry.KeyboardShortcut} action The action to be handled.
  * @return {boolean} True if the action has been handled, false otherwise.
  */
-Blockly.IBlocklyActionable.prototype.onBlocklyAction;
+Blockly.IKeyboardAccessible.prototype.onShortcut;
```

Comment:
It's not clear to me why this was renamed to `onShortcut`.
Related, it might help to reword the JSDoc since it's not longer called on action.